packages/react/src/tabs/localPalette.ts

```diff
@@ -0,0 +1,37 @@
+import { boxPalette } from '../core';
+
+const backgroundMap = {
```

Comment:
@nathsimpson can you confirm these values are correct?

Reply:
We've agreed to diverge from our token conventions for this case, due to the amount of greys needed to achieve the layout.

We've therefore gone in the direction of 'component tokens', which decouples Tabs from our existing schema.

As for the colour values, they reflect my understanding of what we agreed upon this morning 👍